test/fast-list-test.js

```diff
@@ -196,21 +202,23 @@ suite('FastList >', function() {
     var fastList;
 
     setup(function() {
-      this.sinon.useFakeTimers();
-
       fastList = new FastList(source);
-      scheduler.mutation.yield();
-
-      container.scrollTop = 1200; // 2.5 viewports
-      scheduler.attachDirect.yield();
+      return fastList.rendered.then(function() {
+        this.sinon.useFakeTimers();
+        container.scrollTop = 1200; // 2.5 viewports
+        scheduler.attachDirect.yield();
+      }.bind(this));
     });
 
     test('it updates the full rendering', function() {
+      var start = 15;
```

Comment:
I was thinking more in the line of a 
`var start, var end` global to the suite. and then in each `setup`, when we change `scrollTop` we also update them.